<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>651</width>
     <height>521</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/* Fond général de l'interface */
QWidget {
    background-color: #2E2E2E;
    color: white;
    font-family: &quot;Century Gothic&quot;;
    font-size: 18px;
}

/* Cadre principal */
QFrame#frame {
    background: #333;
    border-radius: 15px;
    padding: 15px;
}

/* Titre principal */
QLabel#label_3 {
    font-size: 22px;
    font-weight: bold;
    color: #00FF7F;
    text-align: center;
}

/* Labels des capteurs */
QLabel#label_1, QLabel#label_2 {
    font-size: 16px;
    color: #FFFFFF;
}

/* Valeurs des capteurs */
QLabel#val_temp, QLabel#val_hum {
    font-size: 20px;
    font-weight: bold;
    color: #FFD700; /* Jaune doré */
}

/* Icônes des capteurs */
QToolButton#temp, QToolButton#hum {
    background: transparent;
    border: none;
    font-size: 24px;
}

QToolButton#temp:hover, QToolButton#hum:hover {
    background: rgba(255, 255, 255, 0.2);
    border-radius: 10px;
}

/* Bouton avec logo EcoFit */
QToolButton#toolButton {
    background: transparent;
    border: none;
}
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Capteur humidité :</string>
    </property>
   </widget>
   <widget class="QToolButton" name="temp">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>160</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    background-color: black; /* Fond noir */
}
</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="../supervision_admin/resources.qrc">
      <normaloff>:/images/humidite.png</normaloff>:/images/humidite.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Capteur température :</string>
    </property>
   </widget>
   <widget class="QToolButton" name="hum">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>300</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    background-color: black; /* Fond noir */
}
</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="../supervision_admin/resources.qrc">
      <normaloff>:/images/temperature.jpg</normaloff>:/images/temperature.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>491</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>               SUPERVISION DES CAPTEURS</string>
    </property>
   </widget>
   <widget class="QLabel" name="val_hum">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>160</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="val_temp">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>300</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>0</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton {
    background: transparent;
    border: none;
}
</string>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="icon">
     <iconset resource="../supervision_admin/resources.qrc">
      <normaloff>:/images/ecofit.png</normaloff>:/images/ecofit.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>140</width>
      <height>140</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
